lib/ex_limiter/storage/pg2_shard/supervisor.ex

```diff
@@ -19,6 +19,8 @@ defmodule ExLimiter.Storage.PG2Shard.Supervisor do
     children = [{Router, []} | shards] |> Enum.reverse()
 
     :telemetry.attach_many("exlimiter-metrics-handler", Worker.telemetry_events(), &@telemetry.handle_event/4, nil)
+    
+    {:ok, _pid} = :pg.start_link()
```

Comment:
This creates a global "scope" which all other :pg functions will use by default.

Reply:
does this need to be supervised?

Reply:
Yes.  I'll merge it with the rest of the children